sql/moz-fx-data-shared-prod/org_mozilla_fenix_derived/experiment_search_events_live_v1/init.sql

```diff
@@ -0,0 +1,45 @@
+CREATE MATERIALIZED VIEW `moz-fx-data-shared-prod.org_mozilla_fenix_derived.experiment_search_events_live_v1`
```

Comment:
Can we add commentary to the tops of these files about how the three channels need to stay in sync?